libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1066,11 +1069,17 @@ void GCS_MAVLINK::send_scaled_pressure(AP_Baro &barometer)
 {
     uint32_t now = AP_HAL::millis();
     float pressure = barometer.get_pressure(0);
+    float diff_pressure = 0; // pascal
+
+#if APM_BUILD_TYPE(APM_BUILD_ArduPlane)
```

Comment:
Instead of this I'd prefer that `AP::airspeed` didn't exist and we used the `get_singleton` method to check if it returned nullptr - having a method return a reference to an optional library isn't a very good idea.
Also not only that avoids making the file vehicle dependent but it also makes sure that when airspeed is added to Copter (it has been proposed and I think a PR even exists), we don't miss this.

Reply:
Done this way, consumes 32 bytes more on copter, 8 less on plane.